CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * avifenc, avifdec: Allow "-j all" to automatically use all of the cores on the machine (#670)
 
 ### Changed
-
+* Re-mapping libavif speed 7-9 to libaom speed 7-9 (#682)
```

Comment:
Thanks Wan-Teh. Where would you suggest to put those comments?

Reply:
Since libavif does not have release notes, ideally we should put these instructions in CHANGELOG.md.

@joedrago Joe, Is it okay to add this kind of instructions in CHANGELOG.md?

Reply:
I can make a new `Release Notes` section in Changelog, under `Unreleased`. For now, just add some additional bullets under Changed and we can reorganize before the next release.

Reply:
Done